dev/bazel/release.bzl

```diff
@@ -82,12 +82,38 @@ def _copy_lib(ctx, prefix):
         dst_files.append(dst_file)
     return dst_files
 
+def _copy_env(ctx, prefix):
+    lib_prefix = paths.join(prefix, "env")
+    env_files = env_scripts_generator(ctx)
+    dst_files = []
+    for lib in env_files:
+        dst_path = paths.join(lib_prefix, lib.basename)
+        dst_file = _copy(ctx, lib, dst_path)
+        dst_files.append(dst_file)
+    return dst_files
+
+#draft version for env script, its necessary to do it autogenerated
+def env_scripts_generator(ctx):
```

Comment:
we just need to take vars scripts from here - https://github.com/oneapi-src/oneDAL/tree/main/deploy/local
files have to be renamed on copy